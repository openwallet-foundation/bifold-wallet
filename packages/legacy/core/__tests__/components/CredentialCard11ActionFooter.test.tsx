import '@testing-library/jest-native/extend-expect'
import { render } from '@testing-library/react-native'
import React from 'react'

import CredentialActionFooter from '../../App/components/misc/CredentialCard11ActionFooter'

describe('CredentialCard11ActionFooter Component', () => {
  test('Matches snapshot', async () => {
    const tree = render(<CredentialActionFooter testID={'test'} text={'sample'} onPress={jest.fn()} />)

    expect(tree).toMatchSnapshot()
  })
})
