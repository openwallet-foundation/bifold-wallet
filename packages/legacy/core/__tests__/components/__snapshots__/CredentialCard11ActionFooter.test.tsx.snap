// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CredentialCard11ActionFooter Component Matches snapshot 1`] = `
<View>
  <View
    style={
      Object {
        "backgroundColor": "#D3D3D3",
        "height": 2,
        "marginVertical": 10,
        "width": "100%",
      }
    }
  />
  <View>
    <View
      accessibilityState={
        Object {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        Object {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "flexDirection": "row",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
      testID="com.ariesbifold:id/test"
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontSize": 20,
            "fontWeight": "bold",
          }
        }
      >
        sample
      </Text>
      <Text
        allowFontScaling={false}
        selectable={false}
        style={
          Array [
            Object {
              "color": undefined,
              "fontSize": 12,
            },
            Array [
              Object {
                "color": "#FFFFFF",
                "fontSize": 20,
                "fontWeight": "bold",
              },
              Object {
                "fontSize": 25,
              },
            ],
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        î—Œ
      </Text>
    </View>
  </View>
</View>
`;
